http/vertx/src/main/java/io/cloudevents/http/vertx/package-info.java

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
This is because somehow I got a javadoc-style comment block rather than a plain comment block.

When hitting _"reformat"_ in my IDE then `<p>` tags started to appear, etc 🤣